src/comb/template.clj

```diff
@@ -57,6 +57,6 @@
   ([source]
      (eval source {}))
   ([source bindings]
-     (let [keys (map (comp symbol name) (keys bindings))
+     (let [keys (apply vector (map (comp keyword name) (keys bindings)))
```

Comment:
Sorry, it's my mistake, I didn't realize I tried comb with `1.9.0-alpha14`. It truly work fine in `1.8.0`.